<launch>
    <arg name="camera" default="camera" />
    <arg name="device_name" default="" />
    <arg name="sn" default="" />
    <arg name="vid" default="0" />
    <arg name="pid" default="0" />

    <node pkg="orbbec_camera" type="device_list_node" name="ob_device_list" />

    <include file="$(find orbbec_camera)/launch/include/orbbec.launch">
        <arg name="camera" value="$(arg camera)" />
        <arg name="device_name" value="$(arg device_name)" />
        <arg name="sn" value="$(arg sn)" />
        <arg name="vid" value="$(arg vid)" />
        <arg name="pid" value="$(arg pid)" />
    </include>
</launch>